### PR TITLE
EOS-27348: Updated Iem and MessageBus server cortxconf usage removed

### PR DESCRIPTION
**Pull request by suryakumar1024**
Signed-off-by: suryakumar.kumaravelan <suryakumar.kumaravelan@seagate.com>

# Problem Statement
- Since CortxConf class is no more in use, Need to update all the usage of this class all over the utils code.

# Design
-  Removed the usage in IEM server and MessageServer scripts

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]:  Y
-  Confirm All CODACY errors are resolved [Y/N]:  Y

# Testing 
- [x] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: N
- [x] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: NA
- [x] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [x] Is there a change in filename/package/module or signature [Y/N]: N
- [x] If yes for above point, Is a notification sent to all other cortx components [Y/N]: NA
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/utils/iem_framework/iem_server.py [hunk 1]
```diff
@@ -32,12 +31,12 @@ class IemRequestHandler(MessageServer):
     """ Rest interface of Iem """
 
     @staticmethod
-    def _get_cluster_data(config_path):
-        message_bus_backend = Conf.get('config',\
+    def _get_cluster_data():
+        message_bus_backend = Conf.get(MappedConf._conf_idx,\
```
Comment (ujjwalpl) [inline py-utils/src/utils/iem_framework/iem_server.py:35]: MappedConf must not be used as class instance. Instantiate and keep object reference or use Conf class.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

**Review by rohit-k-dwivedi — APPROVED** [review]
LGTM

**Review by sachinpunadikar — APPROVED** [review]
+1

File: py-utils/src/utils/iem_framework/iem_server.py (hunk 1 above)
Comment (suryakumar1024) [reply py-utils/src/utils/iem_framework/iem_server.py]: Addressed your comment in recent patch.

**Review by ujjwalpl — APPROVED** [review]

**Comment by sachinpunadikar** [comment]
test this please

**Comment by sachinpunadikar** [comment]
retest this please

**Pull request merged**